agent/algorithm/logging.go

```diff
@@ -66,7 +67,7 @@ func (s *Stderr) Write(p []byte) (n int, err error) {
 		s.Logger.Error(string(buf[:n]))
 	}
 
-	if err := s.EventSvc.SendEvent("algorithm-run", "error", json.RawMessage{}); err != nil {
+	if err := s.EventSvc.SendEvent(manager.AlgorithmRun.String(), manager.Error.String(), json.RawMessage{}); err != nil {
```

Comment:
algo run should be in agent

Reply:
If we move this to agent package, importing it here will result in an import cycle hence that will return us to using a string literal.